homeassistant/util/language.py

```diff
@@ -25,10 +25,8 @@ def preferred_regions(
     elif language == "zh":
         if code == "Hant":
             yield "HK"
-        elif code == "Hans":
```

Comment:
How is it expected that `preferred_regions` is used? I think the way it's called from #91521 means a Cantonese speaker may get a voice assistant speaking to them in Mandarin, which is not OK?

Reply:
This is no longer an issue because pipelines store the exact languages 👍